blockchain/blockchain-wallet-v4-frontend
# Feat/send receive usdt

## Description (optional)
Implemented part for send/receive USDT

## Testing Steps (optional)
- login to wallet
- try send flow, make sure that USDT is in list of currencies
